### DISCUSSION TITLE
sending requests via Python3 crashes PS3

### DISCUSSION THREAD

**Discussion by zorua98741** [aldostools/webMAN-MOD#545] [Q&A] [answered]
Hi aldo, I was testing a program I wrote in python3 that utilises webmans webserver, and there seems to be some bug/error that crashes the PS3 while in-game, at seemingly random timings. If at all possible, could you perhaps please provide some insight into what may be going wrong here or how i could provide more information to narrow down the cause?
My best guess is that too many requests are being sent, and this overwhelms the webserver?
```python
import requests
import time
from requests.exceptions import ConnectionError
ip = "PS3 IP address here"
while True:
    try:
        requests.get('http://' + ip)
    except ConnectionError:
        print("\nwMAN server not found, \nCheck connection status of PS3")
    else:
        print("Website exists")
    time.sleep(15)
```

**Comment by aldostools** [comment, marked as answer]
webMAN is a simple web server with no much resources available. It can handle up to 8 concurrent connections. That includes the resources requested from the initial url (e.g. css, javascript, images, frames, etc.)

If the server receives too many requests the server will fail to respond and timeout.
